.circleci/config.yml

```diff
@@ -10,6 +10,9 @@ jobs:
     # this is for the circle 1.0 API
     parallelism: 10
     steps:
+      - add_ssh_keys:
```

Comment:
This probably isn't needed since re-adding the deploy key seems to have worked?

Reply:
removed